### PR TITLE
Add Pablo Mercado (@odacremolbap) as Eventing approver

### PR DESCRIPTION
**Pull request by devguyio**
Pablo Mercado (@odacremolbap) has fulfilled the approver criteria and the Eventing WG would love to nominate him. He's done a great job pushing Eventing across the 1.0 finish line, performing very thorough code review for large and complex PRs and participating in community contact rotation and regular WG calls.

- [x]  Reviewer of the codebase for at least 3 months. (Earliest seen on Oct 2021)
- [x]  Primary reviewer for at least 10 substantial PRs to the codebase [Reviewed 13 PRs](https://github.com/knative/eventing/pulls?q=is%3Apr+commenter%3Aodacremolbap++-label%3Asize%2FS+-label%3Asize%2FXS+-label%3Asize%2FM+-author%3Aodacremolbap).
- [x] Reviewed at least 30 PRs to the codebase. [Reviewed 31 PRs](https://github.com/knative/eventing/pulls?q=is%3Apr+-author%3Aodacremolbap+commenter%3Aodacremolbap+)
- [x] Nominated by an a WG lead - @devguyio 💯  (with no objections from other leads).

### CODE REVIEW

**Comment by lionelvillard** [comment]
/lgtm

congrats @odacremolbap !!

**Comment by evankanderson** [comment]
Can you run `./hack/update-codegen.sh`?

Do you want to update the sandbox org, too?

**Comment by pierDipi** [comment]
+1

**Comment by devguyio** [comment]
I run the generator. I am not sure where exactly in knative-sandbox should I add him.

**Comment by rhuss** [comment]
@devguyio I would expect any new approver to be added to the "Eventing Writers" in https://github.com/knative/community/blob/13c579127342869c6a581097709bafa21d23fe01/peribolos/knative-sandbox.yaml#L613 but it looks like that currently only leads are added here. So you might to sync this section with the similar section in "knative.yaml"

**Comment by devguyio** [comment]
> @devguyio I would expect any new approver to be added to the "Eventing Writers" in
> 
> https://github.com/knative/community/blob/13c579127342869c6a581097709bafa21d23fe01/peribolos/knative-sandbox.yaml#L613
> 
> but it looks like that currently only leads are added here. So you might to sync this section with the similar section in "knative.yaml"

Yes that makes sense. Done!

**Comment by devguyio** [comment]
I don't know what's the problem. @evankanderson what am I missing?

```
{"component":"peribolos","file":"k8s.io/test-infra/prow/cmd/peribolos/main.go:209","func":"main.main","level":"fatal","msg":"Configuration failed: failed to configure knative-sandbox teams: failed to update Eventing Writers child teams: failed to update Source WG Leads metadata: failed to edit knative-sandbox team (Source WG Leads): team.Slug must be populated","severity":"fatal","time":"2022-03-15T09:16:33Z"}
```

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -637,6 +637,17 @@ orgs:
             - devguyio
             - lionelvillard
             privacy: closed
+          Source WG Leads:
```
Comment (rhuss) [inline peribolos/knative-sandbox.yaml:640]: This team does not exist anymore as the Source WG has been merged back into the Eventing WG

Comment (rhuss) [inline peribolos/knative-sandbox.yaml:640]: This team does not exist anymore as the Source WG has been merged back into the Eventing WG

Comment (devguyio) [reply peribolos/knative-sandbox.yaml]: Right! It appeared after rebasing and I didn't notice. Removed.

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

**Comment by rhuss** [comment]
@devguyio @knative/steering-committee we would need a stamp from steering to get this merged.

File: knative-sandbox-OWNERS_ALIASES [hunk 2]
```diff
@@ -112,8 +112,13 @@ aliases:
   - devguyio
   - lionelvillard
   eventing-writers:
+  - aliok
   - devguyio
   - lionelvillard
+  - matzew
+  - odacremolbap
+  - pierDipi
+  - vaikas
```
Comment (pierDipi) [inline knative-sandbox-OWNERS_ALIASES:121]: In this way, every approver in eventing core is an approver in repositories sponsored by Eventing which doesn't seem right.

**Pull request merged**